packages/accordions/src/elements/accordion/components/Panel.tsx

```diff
@@ -55,16 +53,16 @@ const PanelComponent = forwardRef<HTMLElement, HTMLAttributes<HTMLElement>>((pro
 
   return (
     <StyledPanel
-      {...getPanelProps<PanelProps>({
+      {...(getPanelProps({
         role: null,
         ref: mergeRefs([panelRef, ref]),
-        index,
+        value: sectionValue,
         isBare,
         isCompact,
         isExpanded,
         isAnimated,
         ...props
-      })}
+      } as Omit<HTMLAttributes<HTMLElement>, 'role'> & { value: number }) as any)}
```

Comment:
Similar here. Solution is:

```jsx
    <StyledPanel
      isBare={isBare}
      isCompact={isCompact}
      isExpanded={isExpanded}
      isAnimated={isAnimated}
      {...getPanelProps({
        role: role === undefined ? null : 'region',
        ref: mergeRefs([panelRef, ref]),
        value: sectionValue,
        ...props
      }) as HTMLAttributes<HTMLElement>}
    >
```

Change `const panelRef = useRef<HTMLElement>(null);` above and extract `role` from the component props.